include/wx/qt/glcanvas.h

```diff
@@ -85,4 +87,7 @@ class WXDLLIMPEXP_GL wxGLCanvas : public wxGLCanvasBase
     wxDECLARE_CLASS(wxGLCanvas);
 };
 
+#include <QtWidgets/QGestureRecognizer>
+#include <QtWidgets/QGestureEvent>
+
```

Comment:
Sorry to insist, but are those still required here or could they be moved to the `.cpp` file too? We'd rather not include Qt (or any other toolkit/platform) headers from wx public headers, this can result in unexpected symbol clashes.

Reply:
Ok